### DISCUSSION TITLE
In nuxt, how do I custom skins

### DISCUSSION THREAD

**Discussion by xie392** [nuxt/nuxt#16070] [Questions] [answered]
I introduced a global scss in nuxt3. I declared some css variables inside with var, and I styled the body, but it didn't work. Please see the picture below:
![image](https://user-images.githubusercontent.com/61953437/209462862-424e4150-b22b-46e5-ba49-648aae700bb9.png)
![image](https://user-images.githubusercontent.com/61953437/209462873-6cde07cb-b5e1-4b36-a9c5-a9d8022cd3b0.png)
I wonder why the declared css variable failed to set

**Comment by dsvgl** [comment, marked as answer]
try to set your css vars on the `root`:
```css
:root {
  --bg-color: #333;
}
```
